requirements/static

```diff
@@ -0,0 +1,3 @@
+-r base
+-r optional
+mypy
```

Comment:
Why not put mypy in lint as well?

Reply:
lint and static analysis are separate jobs right now, should I combine them? I wanted these files to be composable so if we decide to parallelize more it's easy

Reply:
Forget it